stable/fairwinds-insights/values.yaml

```diff
@@ -339,6 +339,25 @@ databaseCleanupCronjob:
     # -- The user ID to run the database cleanup job under.
     runAsUser: 10324
 
+truncateWorkloadMetrics:
+  # -- Enable truncating workload metrics false by default
+  enabled: false
+  # -- Resources for the truncating workload metrics job.
+  resources:
+    limits:
+      cpu: 250m
+      memory: 512Mi
+    requests:
+      cpu: 40m
+      memory: 32Mi
+  # -- CRON schedules for the truncating workload metrics job.
+  schedules:
```

Comment:
Sorry to add an extra step - but let's make this `[]` by default, and set it in insights-release

Reply:
sure